app/scripts/views/mixins/signed-out-notification-mixin.js

```diff
@@ -7,38 +7,30 @@
 define(function (require, exports, module) {
   'use strict';
 
-  const BaseView = require('views/base');
   const Notifier = require('lib/channels/notifier');
   const Session = require('lib/session');
+  const Url = require('lib/url');
 
   var Mixin = {
     notifications: {
       // populated below using event name aliases
     },
 
     clearSessionAndNavigateToSignIn () {
-      // Unset uid otherwise it will henceforth be impossible
-      // to sign in to different accounts inside this tab.
-      this.relier.unset('uid');
-      this.relier.unset('email');
```

Comment:
Since `email` used to be cleared on the relier, we should probably [*not* pass the email in the query parameters](https://github.com/mozilla/fxa-content-server/pull/4676/files#diff-5a08c710ea753421ee837c5a00857d42R28), otherwise the user's email is going to be displayed on the page, which might not be what the user expects.